modules/openapi-generator/src/test/java/org/openapitools/codegen/swift5/Swift5ClientCodegenTest.java

```diff
@@ -95,6 +95,15 @@ public void testStartingWithNumber() throws Exception {
         Assert.assertEquals(swiftCodegen.toEnumVarName("123EntryName123", null), "_123entryName123");
     }
 
+    @Test(enabled = true)
+    public void testSpecialCharacters() throws Exception {
+        Assert.assertEquals(swiftCodegen.toEnumVarName("1:1", null), "_1Colon1");
```

Comment:
@mbruegmann the previous changes are related to the changes you make in the Swift5ClientCodegen.java.
An easy way to test it is by adding the following test case here that should pass.
```
Assert.assertEquals(swiftCodegen.toEnumVarName(">=", null), "greaterThanOrEqualTo");
```

Reply:
It seems to be an rare edge case. I am doing the replacement character by character now to also replace special characters inside the name. Before only in case of a match of the whole string a replacement was done.
This was the previous code and it would not work for "3=>2" for example. It would only match "=>".

`if (getSymbolName(name) != null) {
            return camelize(WordUtils.capitalizeFully(getSymbolName(name).toUpperCase(Locale.ROOT)), true);
}`
The problem comes from `specialCharReplacements` in `DefaultCodegen`. It's a map with replacements that could be reversed, but the order also is not garanteed.